packages/eslint-plugin-office-addins/test/rules/no-navigational-load.test.ts

```diff
@@ -108,5 +124,23 @@ ruleTester.run('no-navigational-load', rule, {
                 console.log(range.borders.fill);`,
 			errors: [{ messageId: "navigationalLoad", data: { loadValue: "fill" } }]
 		},
+		{
+			code: `
+			  var range = worksheet.getSelectedRange();
+			  range.load("address, font/fill");
+			  await context.sync();
+			  console.log(range.font.fill.color);
+			  console.log(range.address);`,
+			  errors: [{ messageId: "navigationalLoad", data: { loadValue: "font/fill" } }]
+		},
+		{
```

Comment:
What's the difference here?

Reply:
There was no difference. They were duplicated. Deleted one of them.